src/functionalTest/java/uk/gov/hmcts/reform/draftstore/features/security/AuthenticatedRequestTest.java

```diff
@@ -29,4 +40,10 @@ public void should_reject_request_when_missing_required_headers() {
             .assertThat()
             .statusCode(400);
     }
+
+    @Test
+    public void should_accept_authenticated_service_user() {
+        assertThat(draftStoreSteps.listDrafts()).hasSize(0);
+        assertThat(draftStoreSteps.lastResponse().statusCode()).isEqualTo(OK.value());
```

Comment:
Compare this test to existing test above.
it's hard to find any similarities although they're both hitting the same endpoint, the only difference being one adds jwt token headers and the other doesn't.